test/api/unit/models/build.test.js

```diff
@@ -71,14 +72,12 @@ describe('Build model', () => {
 
   describe('.completeJob(message)', () => {
     it('should mark a build errored with a message', (done) => {
-      factory.build().then(build =>
-        build.updateJobStatus({ status: 'error', message: 'this is an error' })
-      ).then((build) => {
+      factory.build().then(build => build.updateJobStatus({ status: 'error', message: 'this is an error' })).then((build) => {
```

Comment:
is this formatting better? i find the former easier to read

Reply:
some of this was probably auto-formatted...